### ISSUE TITLE
Add support for more taxonomies

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress#970]
At the moment, only categories are supported:
![Screen Shot 2021-11-17 at 9 36 29 AM](https://user-images.githubusercontent.com/3868028/142220965-07b7df38-7883-4904-bf8e-4b260229530c.png)

**Comment by olatechpro** [comment]
@stevejburge, what's the proposal here, are we adding all taxonomy filter differently or we're adding them all in same drop down with each under their taxonomy ? I'm asking so i don't work out of context.

**Comment by stevejburge** [comment]
Hi @olatechpro. Can we include "Taxonomies" into a new Settings tab, alongside "Post Types"?
https://github.com/publishpress/PublishPress/issues/971

**Comment by olatechpro** [comment]
@stevejburge currently working on this. Should we extend the option to disable or enable taxonomy to category as well considering the fact it's existing on the filter page already ? (i.e, add category to taxonomy list in menu with option to disable/enable)

**Comment by stevejburge** [comment]
@olatechpro Yes, please. Let's allow users to disable Category if they want to
